DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Validators/NullValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+using FluentValidation.Results;
+
+namespace GitHubExtension.Security.WebApi.Library.Validators
+{
+    sealed class NullValidator<T> : AbstractValidator<T>
```

Comment:
As I understand it's something like template for those entities which are not registered with IValidator interface correct? And the NullValidator will be skipped. But you can forgot implement validator for entity due to behind the scene you already registered all possible entities. Do you this this is a good solution?